src/main/java/by/custom/utilcalculator/controller/MessageRouter.java

```diff
@@ -35,38 +37,60 @@ public SendMessage route(final Update update) {
 
     //we receive message from user, check it and handle it if it is ok here
     private SendMessage getCheckInputMessageAndGetAnswer(final Message message) {
-        String usersMessage = message.getText();
-        String chatID = message.getChatId().toString();
-        String answer;
+        final String usersMessage = message.getText();
+        final String chatID = message.getChatId().toString();
+        final String answer;
         try {
             answer = route(usersMessage, chatID);
-        } catch (UtilsborException e) {
+        } catch (final UtilsborException e) {
             e.printStackTrace();
             return new SendMessage(chatID, getExceptionText(e));
         }
         return new SendMessage(chatID, answer);
     }
 
     private String route(final String usersMessage, final String chatID) throws UtilsborException {
-        String answer;
-        switch (usersMessage) {
-            case Command.START -> {
+        final String answer;
+
+        while (true) {
```

Comment:
We had switch-case here before. 
Now it becomes impossible to use switch-case, so I replaced it by if.
If i use if-else, it is a terrible spagetti-code. So I decided to use while and brakes to avoid too long if-else to get to the SorryMessage: 
![image](https://github.com/Y-Pazniak/custom_calculator_chatbot/assets/73634869/91b3d4da-9041-47f4-bcbf-7f24589fddf9)